roles/oh-my-zsh/default/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+# 'install', 'remove' or 'ignore'
+oh_my_zsh: 'ignore'
```

Comment:
Can you add:

```
oh_my_zsh: 'install'
```
in 

* `host_vars/debian-stretch.yml`: https://github.com/cytopia/ansible-debian/blob/master/host_vars/debian-stretch.yml#L92
* `host_vars/debian-testing.yml`: https://github.com/cytopia/ansible-debian/blob/master/host_vars/debian-testing.yml#L92

The two debian host vars are the generic ones and used for the travis checks.

Reply:
no problem ... required changes are made.